pkg/kp/healthmanager.go

```diff
@@ -94,6 +95,24 @@ const (
 	// parmeter controls how long it takes for clients to notice that health checks have
 	// stopped.
 	SessionTTL = 15 * time.Second
+
+	// MaxWritesPerMinute sets the average number of writes per minute per service
+	// that will be sent to Consul to update health.
+	MaxWritesPerMinute = 1
```

Comment:
For the purposes of replication, this sets a lower bound on how long a deploy can take, since you are required to wait at least 60 seconds from the time a node goes unhealthy (from the old version being stopped) to the new version being brought up. I'd like to raise this number as much as we can tolerate - is 4 okay?